src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -1007,6 +1020,67 @@ public static void deleteDirectoryIfExists(String dirPath) {
         }
     }
 
+    /**
+     * Clean project.
+     */
+    public void stopTerminal() {
+        if (!shouldCleanupTerminal) {
```

Comment:
Modified it to execute the cleanAndResetTerminal() method only if shouldCleanupTerminal is true. Since both tests ( testOpenBuildFileActionUsingPopUpMenu() and testMultipleConfigEditHistory()) where shouldCleanupTerminal is false doesn't require cleanAndResetTerminal().